rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -361,6 +361,8 @@ extern void pre_main (void);
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE / sizeof(uint32_t)];
 #elif defined(TARGET_XDOT_L151CC)
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE * 6 / sizeof(uint32_t)];
+#elif defined(TARGET_REALTEK_RTL8195AM)
```

Comment:
Hi @bulislaw , we found in CI_test, specifically the SPI test in CI test, the main_thread_stack is used up and overflow during the test. We're looking into this issue